src/vmm/src/guest_config/cpuid/amd.rs

```diff
@@ -100,7 +100,7 @@ pub enum ExtendedApicIdError {
 // We use this 2nd implementation so we can conveniently define functions only used within
 // `normalize`.
 #[allow(clippy::multiple_inherent_impl)]
-impl super::AmdCpuid {
+impl super::Cpuid {
     /// We always use this brand string.
     const DEFAULT_BRAND_STRING: &[u8; BRAND_STRING_LENGTH] =
```

Comment:
This constant would not be correct for Intel.

Reply:
Good catch! will fix.